microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -52,6 +52,22 @@ This release includes integration with the greenboot health check framework. Gre
 //==== New feature for use during installation here
 //can include a note about a change in base RHEL image
 
+[id="microshift-4-13-support"]
+=== Support
+
+[id="microshift-4-13-sos-reports"]
+==== {product-title} sos reports
+With this release you can run an `sos` report to collect troubleshooting information about a host. The report will generate a detailed report with data from all enabled plugins and different components and applications in a system.
+
+For more information, see xref:../microshift_support/microshift-sos-report.adoc#about-microshift-sos-reports_microshift-sos-report[About MicroShift sos reports].
+
+[id="microshift-4-13-etcd"]
+==== {product-title} etcd
+
+With this release, {product-title} etcd is run as a separate process whose lifecycle is managed automatically by {product-title}. You can gather `journalctl` logs to observe and debug the etcd server logs.
+
+For more information, see xref:../microshift_support/microshift-etcd.html#microshift-observe-debug-etcd-server_microshift-etcd[MicroShift etcd].
```

Comment:
This link seems to be going to the first module in the assembly: Observe and debug the MicroShift etcd server. Is that the intent? If so, maybe update the link text.